w3sper.js/src/network/components/transactions.js

```diff
@@ -22,6 +22,9 @@ class TransactionExecutedEvent extends RuesEvent {
   }
 
   memo(options = {}) {
```

Comment:
Question, as I never used this, yet: could be useful for the event to have something to retrieve the contract call as well?